### PR TITLE
Social share separation

### PR DESCRIPTION
**Pull request by vrundakansara**
### Description

Social share - Separation of an individual social share as a child block.

### Screenshots

https://abhijitn-bsf.tinytake.com/tt/NDIyODcxMl8xMzIxNTk5Ng

### Types of changes

Breaking change

### How has this been tested?

1. Create Social Share in the master branch.
2. Checkout to this branch and check is there any CSS change occur or not.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by vrundakansara** [comment]
Alignment options are missing - https://share.getcloudapp.com/nOu8WzKG

**Comment by vrundakansara** [comment]
Add this CSS in editor.scss for Social Share.

```.uagb-social-share__wrap > .block-editor-inner-blocks {
    width: 100%;
}```

**Comment by vrundakansara** [comment]
Icon type - Image
https://share.getcloudapp.com/04uPkGRD

Validation error for images.

File: src/blocks/social-share/editor.scss [hunk 1]
```diff
@@ -5,3 +5,22 @@
...
+}
+
+[data-type="uagb/social-share"] .uagb-social-share__layout-horizontal .block-editor-inner-blocks > .block-editor-block-list__layout {
+  display: flex; }
+
+[data-type="uagb/social-share"] .block-editor-inner-blocks > .block-editor-block-list__layout > .wp-block {
+  width: auto;
+  padding-left: 0;
+  padding-right: 0;
+  margin-left: 0;
```
Comment (vrundakansara) [inline src/blocks/social-share/editor.scss:23]: Can we use `margin: 0;` instead?

File: src/blocks/social-share/save.js [hunk 2]
```diff
@@ -18,7 +22,7 @@ export default function save( props ) {
 		socials,
 		social_count,
 		social_layout,
-		current_url
+		// current_url
```
Comment (vrundakansara) [inline src/blocks/social-share/save.js:25]: Unwanted code?

**Comment by RajkiranBagal** [comment]
Layout Horizontal, Stack on Desktop and alignment not working.
 - Backend Screenshot - https://imgur.com/gCGQJ8b
 - In the frontend, It gets stacked but alignment does not work - https://imgur.com/kZ5hAxI

File: src/blocks/social-share-child/edit.js [hunk 3]
```diff
@@ -0,0 +1,309 @@
+/**
+ * BLOCK: Icon List - Edit Class
```
Comment (vrundakansara) [inline src/blocks/social-share-child/edit.js:2]: Change the comment to suit this block.

File: src/blocks/social-share-child/save.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+/**
+ * BLOCK: Column - Save Block
```
Comment (vrundakansara) [inline src/blocks/social-share-child/save.js:2]: Change the comment to suit this block.

File: src/blocks/social-share-child/style.scss [hunk 5]
```diff
@@ -0,0 +1,128 @@
+/**
```
Comment (vrundakansara) [inline src/blocks/social-share-child/style.scss:1]: Do we need this entire CSS? This looks like a repeated code from Social Share parent block.

**Pull request merged**